packages/widget_toolkit_pin/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: widget_toolkit_pin
 description: This package provide out of the box entering PIN code
   functionality, which can be used with biometric authentication.
-version: 0.1.0
+version: 0.1.1
```

Comment:
Similarly to the one above, we should increase the major version because of the changes in `rxdart` and `rx_bloc`.
So this is safe to update to `0.2.0` (changelog version included).